config/settings/production.py

```diff
@@ -91,12 +87,7 @@
     "version": 1,
     "disable_existing_loggers": False,
     "filters": {"require_debug_false": {"()": "django.utils.log.RequireDebugFalse"}},
-    "formatters": {
-        "verbose": {
-            "format": "%(levelname)s %(asctime)s %(module)s "
-            "%(process)d %(thread)d %(message)s"
-        }
-    },
+    "formatters": {"verbose": {"format": "%(levelname)s %(asctime)s %(module)s " "%(process)d %(thread)d %(message)s"}},
```

Comment:
This is an interesting change. This could probably just be this I think?
```suggestion
    "formatters": {"verbose": {"format": "%(levelname)s %(asctime)s %(module)s %(process)d %(thread)d %(message)s"}},
```

Reply:
Agree in principle, but I don't think it needs to be changed in this PR since it's manual. (OK if it is)

Reply:
I think this will get squished out when we run ruff for linting (because it hates `%` for string interpolation), so not worried about it for now. It won't be around long.